src/modules/LoggedIn/Project/ProjectList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react'
-import { Card, Row, Col } from 'antd'
+import { Card, Row, Col, message } from 'antd'
 import { useProjectSync } from './useProjectSync'
+import { Popconfirm } from 'antd';
+import { DeleteOutlined, LeftCircleFilled } from '@ant-design/icons';
+
+const confirm = () => {
+  message.success('Task Deleted');
+}
+
+const cancel = () => {
+  message.error('Canceled');
+}
```

Comment:
method2 kyk gini dimasukin ke componentnya aja. Ntar biar bisa ngakses `useProjectSync`. Sama jangan lupa dikasih memoization pake `React.useCallback`

```suggestion
const confirm = React.useCallback(() => {
  message.success('Task Deleted');
}, [])

const cancel = React.useCallback(() => {
  message.error('Canceled');
}, [])
```